cpp/ql/lib/semmle/code/cpp/security/BufferWrite.qll

```diff
@@ -68,16 +68,37 @@ abstract class BufferWrite extends Expr {
   /**
    * Gets an upper bound to the amount of data that's being written (if one
    * can be found).
+   * DEPRECATED: getMaxData\1 should be used and overridden instead
```

Comment:
I think the syntax is:
```suggestion
   * DEPRECATED: getMaxData/1 should be used and overridden instead
```
And similarly for the other deprecation messages.

Reply:
ok!